build.gradle.kts

```
@@ -9,6 +9,7 @@ plugins {
         alias(detekt) apply false
         alias(kotliner) apply false
     }
+    id("com.google.devtools.ksp") version "1.9.10-1.0.13" apply false
```

Please move this plugin to `libs.versions.toml` instead of hardcoding it here

```suggestion
    alias(ksp) apply false
```

```kotlin
 [plugins]
    ksp = { id = "com.google.devtools.ksp", version.ref = "ksp" }
```